internalv2compat/compat.go

```diff
@@ -80,3 +88,44 @@ func V2TracingEnabled() bool {
 	defer v2Tracing.Unlock()
 	return v2Tracing.enabled
 }
+func SetV2TracingThriftClientMiddleware(middleware thrift.ClientMiddleware) {
```

Comment:
🔕 I would put the customary spaces in between each function and the one above it